### PR TITLE
DBZ-4071 Recipient email address should be a variable in all Jenkins jobs

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-4071

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Thx!

File: jenkins-jobs/pipelines/build-debezium-images.groovy [hunk 1]
```diff
@@ -181,6 +182,6 @@ node('Slave') {
             }
         }
     } finally {
-        mail to: 'jpechane@redhat.com', subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
+        mail to: ${MAIL_ID}, subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
```
Comment (jpechane) [inline jenkins-jobs/pipelines/build-debezium-images.groovy:185]: @ani-sha In all cases you should either use MAIL_ID or `"${MAIL_ID}"` as you don't define it as string now but referes to it as variable in string.

File: jenkins-jobs/pipelines/build-debezium-images.groovy [hunk 2]
```diff
@@ -7,6 +7,7 @@ STREAMS_TO_BUILD_COUNT = STREAMS_TO_BUILD_COUNT.toInteger()
 TAGS_PER_STREAM_COUNT = TAGS_PER_STREAM_COUNT.toInteger()
 GIT_CREDENTIALS_ID = 'debezium-github'
 DOCKER_CREDENTIALS_ID = 'debezium-dockerhub'
+MAIL_ID = 'jpechane@redhat.com'
```
Comment (jpechane) [inline jenkins-jobs/pipelines/build-debezium-images.groovy:10]: @ani-sha This should not be hardcoded by passed as parameter to the job, you should modify job definitions and add the param there with this value as the default one.

Comment (ani-sha) [reply jenkins-jobs/pipelines/build-debezium-images.groovy:10]: Addressed it in https://github.com/debezium/debezium/pull/2765

**Pull request merged**